### ISSUE TITLE
Hana 1909 working but status and title inconsistence

### ISSUE THREAD

**Issue by hansstangl1** [tricktresor/zapcommander#6]
Very nice tool. I could import to hana 1909, but status and title are corrupted/inconsistent (ZAPCMD_SAPCOMMANDER). I can run program. During import I had to select "ignore not fitting component versions".

**Comment by christianguenter2** [comment]
What kind of inconsistencies do you have?
What do you mean with "during import ... ignore not fitting component versions"? That sounds like you imported the transport, right? There have been some fixes which are only in the repo. So I'd suggest you pull the current version from this repo.

**Comment by hansstangl1** [comment]
Hello. I have used files from https://code.google.com/archive/p/sapcommander/downloads because they are cofile and data files. (year 2012). I have not found newer cofile or datafiles which I could import...

**Comment by hansstangl1** [comment]
I think it can be something like that: There has been some status_200 I think. it has been deleted. In object list of the package there is now an object marked for deletion. This objects have problems during import with option ignore not fitting component versions. Maybe new creation of object list before creating the transport would help, but for that you would need a fitting version and not Hana 1909. But that´s just a possibility...

**Comment by christianguenter2** [comment]
I highly recommend to install via abapGit. As the transport is definitely outdated. We only can support you with the current version here in the git repo.

**Comment by hansstangl1** [comment]
I have got no knowledge about abapGit. Import of data and cofiles I had got the rights for that. But for abapGit I would have to setup a whole infrastructure. I have not got the rights for that and project will never be granted. So abapGit is nice for keeping code uptodate but if you have not got any prepared data and cofiles a most of interested people will not be able to use it. Most of the companies I have been split rights between developer and infrastructure. I am not a freelancer having got both rights. Problem of our time: Save installations which can nobody use anymore because of the security restrictions.

**Comment by hansstangl1** [comment]
1)
Interesting. I have found the same site and read it. But I have not found any zip in https://github.com/tricktresor/zapcommander/tree/master/src. But now I have found the drop down menu code. I understand. Interesting. I will try (see 2)
2) 
And in SSL Setup I have read something about certificates. We are not allowed to install any certificates, so I am not sure, wheter I can use the code. Maybe I don´t need that SSL setup in case of offline project zip import...
3) I have put the code ABAPGIT into system per cut and paste...https://raw.githubusercontent.com/abapGit/build/main/zabapgit.abap 
I have to say I did that the first time.
br
Hans
